FileShareClient/Models/Attributes.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace UKHO.FileShareClient.Models
+{
+    public class Attributes : IEquatable<Attributes> 
+    {
+        public Attributes(string key = default, string[] values = default)
+        {
+            Key = key;
+            Values = values;
+        }
+
+        [DataMember(Name = "key", EmitDefaultValue = false)]
+        public string Key { get; set; }
+
+        [DataMember(Name = "values", EmitDefaultValue = false)]
+        public string[] Values { get; set; }
+
+        public override string ToString()
+        {
+            var sb = new StringBuilder();
+            sb.Append("class BatchAttributesSearchResponse {\n");
+            sb.Append("  Key: ").Append(Key).Append("\n");
+            sb.Append("  Values: ").Append(Values).Append("\n");
```

Comment:
Please add a covering unit test to verify the `Values` text is correct. I'm concerned it might print the type name instead of the values in the array